pkg/vfs/dirfs_unit_test.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2022 Chainguard, Inc.
```

Comment:
Idiomatically this should probably be called `dirfs_test.go`

Reply:
Blame me for that pattern, I tend to put the unit tests in a `_unit_test.go` file like that because I usually use `package_test.go` for integration tests (running in a different package, outside of the test subject). Like this:

https://github.com/chainguard-dev/apko/tree/main/pkg/sbom

Sorry (and also happy to do it in another way) :grimacing:

Reply:
Oh okay. I don't feel that strongly, especially if there's precedent elsewhere in this code.